### PR TITLE
Underscore separation is allowed for plugin annotation class properties

### PR DESCRIPTION
**Pull request by JTubex**
PR for issue https://www.drupal.org/project/coder/issues/2804739.

### CODE REVIEW

**Comment by klausi** [comment]
Tests are failing because of coding standards. Can you fix them?

**Comment by JTubex** [comment]
Coding standards are fixed and all tests are passing.

File: coder_sniffer/Drupal/Test/NamingConventions/ValidVariableNameUnitTest.inc [hunk 1]
```diff
@@ -2,3 +2,25 @@
...
+class TestAnnotationProperties2 extends ViewsPluginAnnotationBase {
+
+  /**
+   * The entity_type.
+   *
+   * @var string
+   */
+  public $entity_type;
+
+}
```
Comment (klausi) [inline coder_sniffer/Drupal/Test/NamingConventions/ValidVariableNameUnitTest.inc:26]: we are missing a test case for AnnotationInterface, right?

**Review by klausi — CHANGES_REQUESTED** [review]

File: coder_sniffer/Drupal/Test/NamingConventions/ValidVariableNameUnitTest.inc (hunk 1 above)
Comment (JTubex) [reply coder_sniffer/Drupal/Test/NamingConventions/ValidVariableNameUnitTest.inc:26]: You're right. I added a test case for AnnotationInterface.

**Pull request merged**